product_supplierinfo_archive/views/product_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_supplierinfo_tree_view" model="ir.ui.view">
+        <field
+            name="name"
+        >product.supplierinfo.tree.view - product_supplierinfo_archive</field>
+        <field name="model">product.supplierinfo</field>
+        <field name="inherit_id" ref="product.product_supplierinfo_tree_view" />
+        <field name="arch" type="xml">
+            <field name="price" position="after">
+                <field name="active" invisible="1" />
```

Comment:
> Missing fields used in domains and other attributes are added to the view as invisible automatically. If you encounter fields in views with invisible="True", they can probably be removed. Reference: https://github.com/odoo/odoo/pull/137031

Reply:
looks weird

![image](https://github.com/user-attachments/assets/79cc7471-4659-405b-a0bf-6a724f509afd)

Reply:
@pfranck can you address this comment please? :pray: